<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionEditDialog</class>
 <widget class="QDialog" name="ConnectionEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="loDriverName">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblDriverName">
       <property name="text">
        <string>Driver name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbDriverName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="loAlias">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblAlias">
       <property name="text">
        <string>Alias</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtConnectionName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="loDatabaseName">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblDatabaseName">
       <property name="text">
        <string>Database name/Local path</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="loDatabaseEditor">
       <item>
        <widget class="QLineEdit" name="edtDatabaseName"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowseLocalAddress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="loHostAddress">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblHostAddress">
       <property name="text">
        <string>Host address</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtHostAddress"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="loUserName">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblUsername">
       <property name="text">
        <string>Username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtUserName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="loPassword">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtPassword">
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loBottomPanel">
     <item>
      <widget class="QPushButton" name="btnTryToConnect">
       <property name="text">
        <string>Test connection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDrop">
       <property name="text">
        <string>Drop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
